app/components/subcase/subcase-titles-panel/subcase-titles-panel-view.js

```diff
@@ -17,7 +18,26 @@ export default class SubcaseTitlesPanelView extends Component {
 
   @task
   *loadApproved() {
-    this.approved = yield this.args.subcase.approved;
+    const meeting = yield this.args.subcase.requestedForMeeting;
```

Comment:
Maybe this should be moved to a util, but again the store is needed and it feels weird to pass in the store for fetching. Input welcome.

Reply:
Moved to a service